### PR TITLE
Don't treat mts/cts files as config files by using same check as dt-tools

### PR DESCRIPTION
**Pull request by jakebailey**
Noticed on https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68772 that cts/mts files are being considered config files. That's not right.

### CODE REVIEW

File: package-lock.json
Comment (jakebailey) [inline package-lock.json:1]: This adds 1000 lines to the package-json; I can copy the glob out instead if that feels better. I doubt it will change.

File: package-lock.json [hunk 1]
```diff
@@ -1052,11 +1053,103 @@
         "parsimmon": "^1.13.0"
       }
     },
+    "node_modules/@definitelytyped/typescript-packages": {
```
Comment (jakebailey) [inline package-lock.json:1056]: I have no idea why this dep is being pulled in. Maybe I will extract the function.

Comment (jakebailey) [reply package-lock.json:1056]: Sent https://github.com/microsoft/DefinitelyTyped-tools/pull/964

File: package-lock.json
Comment (jakebailey) [reply package-lock.json:1]: Slightly better after refactoring dt-tools, but still pulling in stuff. Maybe that's okay.

File: src/_tests/fixtures/67090/result.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
...`](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67090/files/bb7d42f514b9f9cfca6b9a84dc99fac76ac35643#diff-9f471f9ba6ffa73024ae9aeb049dd98078364c565db83ee831e0498cbb678d67): edited\n* `marked-terminal` — [on npm](https://www.npmjs.com/package/marked-terminal), [on unpkg](https://unpkg.com/browse/marked-terminal@latest/)\n  - Config files to check:\n    - [`marked-terminal/marked-terminal-tests.cts`](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67090/files/bb7d42f514b9f9cfca6b9a84dc99fac76ac35643#diff-c57707862a9de137333983d3c0ee294d8a046f4f23a7e2557f7c4f4c77529dd7): edited\n\n## Code Reviews\n\nThis PR adds a new definition, so it needs to be reviewed by a DT maintainer before it can be merged.\n\nYou can test the changes of this PR [in the Playground](https://www.typescriptlang.org/play/?dtPR=67090&install-plugin=playground-dt-review).\n\n## Status\n\n * ✅ No merge conflicts\n * ❌ Continuous integration tests have failed\n * 🕐 Only a DT maintainer can approve changes when there are new packages added\n\nOnce every item on this list is checked, I'll ask you for permission to merge and publish the changes.\n\n----------------------\n... diagnostics scrubbed ..."
```
Comment (andrewbranch) [inline src/_tests/fixtures/67090/result.json:12]: Looks like we need to account for non-declaration test files too:


> - Config files to check:\n    - [`marked-terminal/marked-terminal-tests.cts`]

Comment (jakebailey) [reply src/_tests/fixtures/67090/result.json:12]: Aha, yes.

Comment (jakebailey) [reply src/_tests/fixtures/67090/result.json:12]: There's even more weird edge cases here... may do a little refactoring to fix them.

**Review by andrewbranch — APPROVED** [review]

File: src/pr-info.ts [hunk 3]
```diff
@@ -341,21 +342,16 @@ async function getPackageInfosEtc(
...
-        case "md": return [pkg, { path, kind: "markdown" }];
-        default: {
-            const contentGetter = (oid: string) => async () => fetchFile(`${oid}:${path}`);
-            const suspect = await configSuspicious(path, contentGetter(newId), contentGetter(oldId));
-            return [pkg, { path, kind: suspect ? "package-meta" : "package-meta-ok", suspect }];
-        }
-    }
+
+    if (isDeclarationPath(path)) return [pkg, { path, kind: "definition" }];
+    if (/\.[cm]?tsx?$/.test(path)) return [pkg, { path, kind: "test" }];
```
Comment (RyanCavanaugh) [inline src/pr-info.ts:349]: Any concern that this allows `mtsx` ? Seems benign but I have to ask

Comment (jakebailey) [reply src/pr-info.ts:349]: I guess it does, but something else will error. But, I'll fix the regex.

Comment (jakebailey) [reply src/pr-info.ts:349]: #489
